bin/prepare_project_edition.sh

```diff
@@ -16,7 +16,10 @@ if [[ -n "${DOCKER_PASSWORD}" ]]; then
 fi
 
 # Create container to install dependencies
-docker run --name install_dependencies -d --volume=${PROJECT_BUILD_DIR}:/var/www:cached --volume=${HOME}/.composer:/root/.composer -e APP_ENV -e APP_DEBUG ${PHP_IMAGE}
+docker run --name install_dependencies -d \
+--volume=${PROJECT_BUILD_DIR}:/var/www:cached --volume=${HOME}/.composer:/root/.composer \
+-e APP_ENV -e APP_DEBUG -e PHP_INI_ENV_memory_limit -e COMPOSER_MEMORY_LIMIT \
+${PHP_IMAGE}
```

Comment:
`COMPOSER_MEMORY_LIMIT` and `PHP_INI_ENV_memory_limit` variables are added, they are needed for the `composer install`/`cache:clear` phases (especially in commerce)